src/Volume/Volume.make

```diff
@@ -37,6 +37,7 @@ endif
 
 ifeq "$(ENABLE_WOLFCRYPT)" "0"
 ifeq "$(PLATFORM)" "MacOSX"
+ifeq "$(COMPILE_ASM)" "true"
```

Comment:
`COMPILE_ASM` variable needs to be also set somewhere else than only in `build_veracrypt_macosx.sh`, like in `Makefile`, or else normal make will not work properly as it is not defined without this build script.

Reply:
Thanks for the review @Jertzukka! 

Good call and find. I addressed this in 3e5168f1a6ceb9808aee8435a9a96ed15841797f, moving the check `ifneq` instead `ifeq` so the undefined variable doesn't fail the build. I'll try follow up on the other issues you mentioned tomorrow.